api/v1beta1/kustomization_types.go

```diff
@@ -92,6 +93,11 @@ type KustomizationSpec struct {
 	// +optional
 	Timeout *metav1.Duration `json:"timeout,omitempty"`
 
+	// Images override image definitions in manifests
+	// It can set the same configuration parameters with kustomization.yaml
+	// +optional
+	Images []kustypes.Image `json:"images,omitempty"`
```

Comment:
We need to define our own Image type and not make the API package depend on kustomize.